frontend/integration-tests/views/dashboard-shared.view.ts

```diff
@@ -0,0 +1,4 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { untilNoLoadersPresent } from '../views/crud.view';
+
+export const dashboardIsLoaded = () => browser.wait(until.and(untilNoLoadersPresent)).then(() => browser.sleep(1000));
```

Comment:
what is the reason for the sleep?  + can we extract the time to some named constant?

Reply:
core integration-tests define this in a similar way https://github.com/openshift/console/blob/4add1e0a27ec21af3ff460d657852f1fba36f3fe/frontend/integration-tests/views/crud.view.ts#L21
I presume that the reason for this is that when the loader disappears, the page may not be completely rendered immediately after that, and since the core `isLoaded` method is used on different pages, it's difficult to know what element to wait for...

Reply:
ok

Reply:
> what is the reason for the sleep? + can we extract the time to some named constant?

Do you think we should use the same constant in dashboardIsLoaded and isLoaded functions?

Reply:
I would make it dashboard specific only so it might be tweaked accordingly later. `IsLoaded` is used in many other places.